src/sentry/static/sentry/app/components/dropdownAutoCompleteMenu.jsx

```diff
@@ -380,6 +380,8 @@ class DropdownAutoCompleteMenu extends React.Component {
           // emptyHidesInput is set to true.
           const showInput = !hideInput && (hasItems || !emptyHidesInput);
 
+          const itemCount = autoCompleteResults.filter(i => !i.groupLabel).length;
```

Comment:
If you had 10 results that included 2 `groupLabel`s, wouldn't this just mean that the last 2 items wouldn't be selectable?

Reply:
Disregard, it's because `groupLabels` do not get added to `AutoComplete.items` because of `this.renderRow()` where only autocomplete items get called with `getItemProp` render prop.

Reply:
So I would add an inline comment here about why we're doing this (virtualization).

Additionally, do we only need to calc this when having virtualized rows?